baram/sh_manager.py

```diff
@@ -0,0 +1,30 @@
+import boto3
+import fire
+import pygsheets
+import pandas as pd
+
+
+class SHManager(object):
+    def __init__(self):
+        self.cli = boto3.client('securityhub')
+
+    def list_findings(self):
+        response = self.cli.get_findings(Filters={'WorkflowStatus': [{'Value': 'NEW', 'Comparison': 'EQUALS'}]},
+                                         MaxResults=100)
+        findings = response['Findings']
+        return findings
+
+    def export_to_gsheet(self, service_file: str):
```

Comment:
Maybe the word `gsheet` isn't familiar to people who do not know history of our project. How about using full naming like `google_sheet` instead?

Reply:
agree with your suggestion. will reflect in next commit

Reply:
agree with your suggestion. will reflect it in next commit